samples/matter/lock/boards/nrf54h20dk_nrf54h20_cpuapp.conf

```diff
@@ -5,7 +5,6 @@
 #
 
 # TODO: Workaround to be removed once DFU and external flash will be supported on nRF54H20.
-CONFIG_CHIP_OTA_REQUESTOR=n
 CONFIG_CHIP_QSPI_NOR=n
```

Comment:
I'm not sure whether it is needed or not. Maybe we can remove it all because the external flash will be driven by EXMIF. We can do it here, and within the PR for External flash support as well.

Reply:
Yeah, I assumed it's related to external flash, so I will leave it until we claim support for external flash.